### PR TITLE
write logs one line at a time, instead of chunks at a time

### PR DESCRIPTION
**Pull request by deitch**
As requested by @giggsoff . All of our other logs write logs entry by entry; this just brings the original log-writing inline.

Since all of our drivers expect byte slices, it shouldn't matter.

I validated file and memory, and redid _looks_ right to me, but it would be helpful if @giggsoff would validate. If you find issues, comment here; if not, approve and I will merge in.

### CODE REVIEW

File: pkg/server/apiHandler.go [hunk 1]
```diff
@@ -350,15 +348,14 @@ func (h *apiHandler) logs(w http.ResponseWriter, r *http.Request) {
 			log.Printf("failed to marshal protobuf message into json: %v", err)
 			http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
 		}
-		buf.Write(entryBytes)
+		err = h.manager.WriteLogs(*u, entryBytes)
+		if err != nil {
+			log.Printf("Failed to write logbundle message: %v", err)
```
Comment (giggsoff) [inline pkg/server/apiHandler.go:353]: Just small remark: there is not logbundle here (logbundle->fulllogentry).

Comment (deitch) [reply pkg/server/apiHandler.go]: Good point. Fixed. Anything else?

**Comment by giggsoff** [comment]
Seems, that we have protojson [here](https://github.com/deitch/adam/blob/write-logs-by-line/pkg/server/apiHandler.go#L346) and proto [here](https://github.com/deitch/adam/blob/b6d8baa3495b0f9d6352a627602cc565d572613d/pkg/server/apiHandler.go#L398). I am confused.

**Comment by deitch** [comment]
[This one](https://github.com/deitch/adam/blob/b6d8baa3495b0f9d6352a627602cc565d572613d/pkg/server/apiHandler.go#L398) is wrong (the one in `newLogs()`). It uses `proto.Marshal()` which converts it to the wire format, rather than the json format. We should (and everywhere else do) use the very efficient but specific wire format for on the wire between device and adam (i.e. `apiHandler`), and the less efficient but more universal json format for storage in the driver. 

That `newLog()` one should be changed. But that is beyond the scope of this PR; I will open an issue on it. This one is narrowly focused on writing one line of json at a time to the driver.

Within that scope, any issues?

**Review by giggsoff — APPROVED** [review]
Looks good, thanks for the comments.

**Comment by deitch** [comment]
I am happy to have you checking this @giggsoff .

I opened #53 for the issue you raised.

**Comment by deitch** [comment]
@rvs I can merge this, but need your approval on the PR.

**Pull request merged**